funnel/assets/js/gettext.js

```diff
@@ -101,11 +98,13 @@ class Gettext {
         console.error("Plural forms don't match");
       }
 
+      var msgstr = '';
       if (num <= 1) {
-        return msgidCatalog[1].replace('%d', num);
+        msgstr = sprintf(msgidCatalog[1], { num: num });
       } else {
-        return msgidCatalog[2].replace('%d', num);
+        msgstr = sprintf(msgidCatalog[2], { num: num });
```

Comment:
Isn't this msgidCatalog[0]

Reply:
`[0]` is `msgid_plural`. Check the comments on the top.